src/api/app/controllers/webui/attribute_controller.rb

```diff
@@ -25,6 +23,8 @@ def new
 
     authorize @attribute, :create?
 
+    @attribute_types = AttribType.includes(:attrib_namespace).all.sort_by(&:fullname)
+
     switch_to_webui2
```

Comment:
Hm, why keep this around?

Reply:
It's loading the old assets if I remove it.

Reply:
We still need this, because the new layout is not integrated.